data/hai/hai reveal 5 philosophers.txt

```diff
@@ -570,11 +568,11 @@ mission "Hai Rescue [B07] Return"
 			`	"The people have an intense dislike of Alphas, so they've definitely shown up at some point. The pirates say Alphas will take over any organization they join. They think that aliens would do the same, so Devil-Run pirates are only humans."`
 				goto questions
 			label old
-			`	"Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but the locals claim they're descendants of anarchists from Freedom. There's also no way a city this size could have been built in only fifty years; it must be at least a hundred years old.`
+			`	"Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place so I've had a good chance to talk to some of the locals. They claim that for the most part they're descendants of anarchists from Freedom. There's also no way a city this size could have been built in only fifty years; it must be well over that.`
```

Comment:
```suggestion
			`	"Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place, so I've had a good chance to talk to some of the locals. They claim that for the most part they're descendants of anarchists from Freedom. There's also no way a city this size could have been built in only fifty years; it must be well over that.`
```

Reply:
I question this one.

Reply:
Two independent clauses, need a coordinating conjunction (`so`) _and_ a comma. The alternative would be separating the two sentences entirely:
```suggestion
			`	"Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place. I've had a good chance to talk to some of the locals. They claim that for the most part they're descendants of anarchists from Freedom. There's also no way a city this size could have been built in only fifty years; it must be well over that.`
```
I guess there's an argument that when spoken aloud, it wouldn't have the "pausing" of a comma/period, but then the disadvantage would be having a very long sentence without many breaks: `Our intel believed the city was established fifty years ago when Base Principles and their partners were supposed to have been scoured from Syndicate space, but I'm free to access large parts of this place so I've had a good chance to talk to some of the locals.`
Ultimately up to you.

Reply:
What if we did the long sentence and then interrupted the paragraph with a "he pauses for breath" type deal?
I think something like that would also add to the vibe that he's trying to communicate as much as possible in a short window of time.